MobileBuy/buy/src/main/java/com/shopify/buy/dataprovider/ProductService.java

```diff
@@ -111,59 +112,63 @@
     Observable<List<Product>> getProducts(List<Long> productIds);
 
     /**
-     * Fetch the list of Products in a Collection using the sort order defined in the shop admin
+     * Fetch a page of collections
```

Comment:
Normally, we should not mix functional changes with basic file reorganization changes. It makes it harder to tell what changes have consequences, and which are superficial.